config/README.md

```diff
@@ -166,6 +167,226 @@ fmt.Println(m.World)
 Note that any fields you wish to deserialize into must be exported, just like
 `json.Unmarshal` and friends.
 
+## Environment Variables
+
+YAML provider supports accepting values from the environment.
+For example, consider the following YAML file:
+
+```yaml
+modules:
+  http:
+    port: ${HTTP_PORT:3001}
+```
+
+Upon loading file, YAML provider will look up the HTTP_PORT environment variable
+and if available use it's value. If it's not found, the provided `3001` default
+will be used.
+
+## Command Line Arguments
+
+Command line provider is a static provider that reads flags passed to a program and
+wraps them in the `Provider` interface. It uses dots in flag names as separators
```

Comment:
I think dotted notation is defined on line 112, but yeah, commas could be made more clear. Please, help!